src/_locales/fr/messages.json

```diff
@@ -183,7 +183,7 @@
     "hash": "ba0f60f1ae0c6d6785d5eb410247eb11"
   },
   "translatorCredit": {
-    "message": "Cette extension a été traduite en français par $TRANSLATORS$",
+    "message": "Cette extension a été traduite en français par <a href=\"https://github.com/rugk/mastodon-simplified-federation/blob/master/CONTRIBUTORS.md#french\">@PoorPocketsMcNewHold et @le-jun</a>",
```

Comment:
This won't work, HTML is not allowed.
Also you need to keep the placeholder, don't remove it, it will be replaced with the stuff you put in below.

See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translations for how these placeholders are translated, especially [the official doc](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales).

Reply:
*breathes in deeply* Ok. This ought to do this time or I’m gonna jump out the window

Reply:
well… don't do that :stuck_out_tongue_winking_eye: